ntfysh_client/MainForm.cs

```diff
@@ -361,7 +397,7 @@ private void LoadSettings()
             Program.Settings = settings;
             
             //Check the settings revision. If it is older than the current latest revision, apply the settings defaults missing from previous revision
-            if (Program.Settings.Revision < defaultSettings.ReconnectAttempts)
+            if (Program.Settings.Revision < defaultSettings.Revision)
```

Comment:
I noticed this while debugging.... while I didn't do a proper [Chesterton's Fence investigation](https://en.wikipedia.org/wiki/G._K._Chesterton#Chesterton's_fence) on it, it seem pretty wrong...., I'll dig into history on this more later.

Reply:
Thaaaat looks like a keyboard ninja tab tab enter move typo 🤣 Was that me? Whoops!

Reply:
I'm going to mark this as resolved.